### PR TITLE
Fix code analysis error

### PR DESCRIPTION
**Pull request by Pilchie**
Not sure why I'm seeing this in VS (even when building), but not on the command line.

### CODE REVIEW

File: src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void InitializesIsEnabled(bool? enabled)
         {
             var configuration = new ConfigurationBuilder().AddInMemoryCollection(new[]
             {
-                new KeyValuePair<string, string>("IsEnabledKey", Convert.ToString(enabled))
+                new KeyValuePair<string, string>("IsEnabledKey", Convert.ToString(enabled, null))
```
Comment (JamesNK) [inline src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs:24]: TIL that null is an option here. I've always passed `Culture.InvariantCulture`.

**Review by JamesNK — APPROVED** [review]

**Comment by JamesNK** [comment]
> Not sure why I'm seeing this in VS (even when building), but not on the command line.

Is editorconfig defined code analysis only checked in VS? https://github.com/dotnet/aspnetcore/blob/76d5c625ec702ee7a4a642b52b7ecfd8fc435338/.editorconfig#L72-L74

File: src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs (hunk 1 above)
Comment (dougbu) [reply src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs:24]: Looks like dotnet/runtime goes a bit deeper, to `NumberFormatInfo` values e.g.
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.Extensions/tests/System/Convert.ToString.cs#L485

Suggest `NumberFormatInfo.InvariantInfo` here

Comment (Pilchie) [reply src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs:24]: It goes through https://github.com/dotnet/runtime/blob/e5dd7150e6ced783159a8ae3adb77b899b1204db/src/libraries/System.Private.CoreLib/src/System/Convert.cs#L1942, which means it doesn't matter.  Going to leave it null

**Comment by Pilchie** [comment]
> Is editorconfig defined code analysis only checked in VS?

Shouldn't be, unless it's somehow overriden somewhere.

**Comment by BrennanConroy** [comment]
Code was changed to `ToString()` recently
https://github.com/dotnet/aspnetcore/blob/1048a02802897ca86de536664db3a30f5b4a9ed7/src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs#L24
Does this analysis error still apply now?

**Comment by Pilchie** [comment]
They actually fixed the analyzer too so that it knows that `bool?` is invariant.  No need for this PR.

**Pull request closed without merging**